test/hdnode.js

```diff
@@ -368,6 +405,30 @@ describe('HDNode', function () {
       assert.throws(function () {
         master.derive('foo')
       }, /Expected UInt32/)
+      assert.throws(function () {
+        master.derivePath()
+      }, /Expected Path/)
+      assert.throws(function () {
+        master.derivePath(2)
+      }, /Expected Path/)
+      assert.throws(function () {
+        master.derivePath([2, 3, 4])
+      }, /Expected Path/)
+      assert.throws(function () {
+        master.derivePath('/')
+      }, /Expected Path/)
+      assert.throws(function () {
+        master.derivePath('m/m/123')
+      }, /Expected Path/)
```

Comment:
See https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/fixtures/hdnode.json#L195 for examples